apps/api/src/domain/models/audio-visual/audio-item/queries/audio-item.view-model.event-sourced.ts

```diff
@@ -85,6 +90,14 @@ export class EventSourcedAudioItemViewModel extends BaseEventSourcedResourceView
             this.transcript = Transcript.buildEmpty();
             this.text = '';
         }
+
+        if (Array.isArray(notes))
+            this.notes = notes.map((n) => NoteRecordForResourceViewModel.fromDto(n));
+
+        if (Array.isArray(connections))
```

Comment:
I'm on the fence about a base query repository, but I am pretty convinced we should have a shallow inheritance hierarchy that includes a base resource view model (as we did for the state-persisted models) in the near future.